documentation/staging/content/quickstart/summary.md

```diff
@@ -0,0 +1,369 @@
+       value: "domain1"
+     - name: JAVA_OPTIONS
+       value: "-Dweblogic.StdoutDebugEnabled=false"
+     - name: USER_MEM_ARGS
+       value: "-Djava.security.egd=file:/dev/./urandom -Xms256m -Xmx512m "
+     resources:
+       requests:
+         cpu: "250m"
+         memory: "768Mi"
+
+     # Optional volumes and mounts for the domain's pods. See also 'logHome'.
+     #volumes:
+     #- name: weblogic-domain-storage-volume
+     #  persistentVolumeClaim:
+     #    claimName: sample-domain1-weblogic-sample-pvc
+     #volumeMounts:
+     #- mountPath: /shared
+     #  name: weblogic-domain-storage-volume
+
+   # The desired behavior for starting the domain's administration server.
+   adminServer:
+     # The serverStartState legal values are "RUNNING" or "ADMIN"
+     # "RUNNING" means the listed server will be started up to "RUNNING" mode
+     # "ADMIN" means the listed server will be start up to "ADMIN" mode
+     serverStartState: "RUNNING"
+     # Setup a Kubernetes node port for the administration server default channel
+     #adminService:
+     #  channels:
+     #  - channelName: default
+     #    nodePort: 30701
+
+   # The number of managed servers to start for unlisted clusters
+   replicas: 1
+
+   # The desired behavior for starting a specific cluster's member servers
+   clusters:
+   - clusterName: cluster-1
+     serverStartState: "RUNNING"
+     serverPod:
+       # Instructs Kubernetes scheduler to prefer nodes for new cluster members where there are not
+       # already members of the same cluster.
+       affinity:
+         podAntiAffinity:
+           preferredDuringSchedulingIgnoredDuringExecution:
+             - weight: 100
+               podAffinityTerm:
+                 labelSelector:
+                   matchExpressions:
+                     - key: "weblogic.clusterName"
+                       operator: In
+                       values:
+                         - $(CLUSTER_NAME)
+                 topologyKey: "kubernetes.io/hostname"
+     # The number of managed servers to start for this cluster
+     replicas: 2
+
+   # Change the restartVersion to force the introspector job to rerun
+   # and apply any new model configuration, to also force a subsequent
+   # roll of your domain's WebLogic Server pods.
+   restartVersion: '1'
+
+   # Changes to this field cause the operator to repeat its introspection of the
+   #  WebLogic domain configuration.
+   introspectVersion: '1'
+
+     # Secrets that are referenced by model yaml macros
+     # (the model yaml in the optional configMap or in the image)
+     #secrets:
+     #- sample-domain1-datasource-secret
+
+    {{% /expand %}}
+
+    b. **NOTE**: Instead of running the previous `kubectl apply` command, you can download the WLS Domain YAML file to `/tmp/quickstart/quick-start-domain-resource.yaml` or similar, using the following command, and then apply the file using `kubectl apply -f /tmp/quickstart/quick-start-domain-resource.yaml`:
```

Comment:
Oops - we both missed this:  This says "NOTE: Instead of running the previous kubectl apply command..." but there is no previous apply command.
to fix: combine the note into the "a" bullet, and change wording to "Alternatively, you can download the WLS Domain YAML file to /tmp/quickstart/quick-start-domain-resource.yaml using the following command:"